README.md

```diff
@@ -12,6 +12,43 @@ Simply run
 kurtosis run github.com/kurtosis-tech/etcd-package
 ```
 
+#### Configuration
+
+<details>
+    <summary>Click to see configuration</summary>
+
+You can configure this package using a JSON structure as an argument to the `kurtosis run` function. The full structure that this package accepts is as follows, with default values shown (note that the `//` lines are not valid JSON and should be removed!):
+
+```javascript
+{
+    // The name to give the new etcd service
+    "etcd_name": "etcd",
```

Comment:
Prefix the arg names with the package name to avoid name clashing when the args dictionary is passed to other packages.